src/vue/pages/VerificationGeneral.vue

```diff
@@ -123,7 +123,8 @@
         <div class="app__form-actions">
           <form-confirmation
             v-if="formMixin.isConfirmationShown"
-            @ok="hideConfirmation() || submit()"
+            :is-pending="isFormSubmitting"
```

Comment:
please make sure you apply the changes requested in `VerificationCorporate` also here

Reply:
Ya, tested with another user

Reply:
but I was asking to not forgot bring the same changes from `VerificationGeneral` if they present after the review section